repository-data/application/src/main/resources/hcm-config/configuration/update/WSP-215-add-searchable-type.groovy

```diff
@@ -0,0 +1,47 @@
+package org.hippoecm.frontend.plugins.cms.admin.updater
+
+import org.hippoecm.repository.util.JcrUtils
+import org.onehippo.repository.update.BaseNodeUpdateVisitor
+
+import javax.jcr.Node
+
+/**
+ * Adds searchableType true on each document.
+ */
+class SearchableType extends BaseNodeUpdateVisitor {
+
+    boolean doUpdate(Node node) {
+
+        // Query returns the hippo:handle node for the document
+        // (which has the 3 variants)
```

Comment:
To be precise, the query will return the variants themselves not the handle.

Reply:
I have updated the comment.